src/Protocol/Models/CompletionList.cs

```diff
@@ -9,14 +9,22 @@ namespace OmniSharp.Extensions.LanguageServer.Protocol.Models
     /// Represents a collection of [completion items](#CompletionItem) to be presented
     /// in the editor.
     /// </summary>
-    public class CompletionList
+    public class CompletionList : Container<CompletionItem>
```

Comment:
This is a breaking change unfortunately, but it isn't terrible... completion list should be supported as a simple list.